package/MDAnalysis/analysis/encore/covariance.py

```diff
@@ -215,7 +215,7 @@ def covariance_matrix(ensemble,
         # Select the same atoms in reference structure
         reference_atom_selection = reference.select_atoms(
             ensemble.get_atom_selection_string())
-        reference_coordinates = reference_atom_selection.atoms.coordinates()
+        reference_coordinates = reference_atom_selection.atoms.positions
```

Comment:
The failed codecov check is indicating that the this line isn't covered by a test. This explains why tests didn't identify that `atoms.coordinates()` no longer existed.

If possible could you add a test that checks this code path?